src/main/java/Duke.java

```diff
@@ -1,10 +1,109 @@
+import java.util.Scanner;
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello, my name is Duke! \n What can I do for you?");
+
+        Scanner myObj = new Scanner(System.in);
+
+        Task[] todoList = new Task[100];
+        String userInput;
+        String subUserInput;
+        int markedValue = 0;
+        boolean bye = false;
```

Comment:
The boolean variable bye can be renamed to isProgramFinished or something similar for ease of understanding by the reader.